Dockerfile

```diff
@@ -1,19 +1,27 @@
-# a minimal Nginx container including containerpilot and a simple virtulhost config
+# A minimal Nginx container including ContainerPilot and a simple virtualhost config
 FROM nginx:latest
 
-# install curl
-RUN apt-get update && \
-    apt-get install -y \
-    curl \
-    unzip && \
-    rm -rf /var/lib/apt/lists/*
+# Add some stuff via apt-get
+RUN apt-get update \
+    && pt-get install -y --no-install-recommends \
+        curl \
+        unzip \
+    && rm -rf /var/lib/apt/lists/*
 
-RUN curl -Lo /tmp/consul_template_0.11.0_linux_amd64.zip https://github.com/hashicorp/consul-template/releases/download/v0.11.0/consul_template_0.11.0_linux_amd64.zip && \
-    unzip /tmp/consul_template_0.11.0_linux_amd64.zip && \
-    mv consul-template /bin
+# Add Consul template
+# Releases at https://releases.hashicorp.com/consul-template/
+ENV CONSUL_TEMPLATE_VERSION 0.14.0
+ENV CONSUL_TEMPLATE_SHA1 7c70ea5f230a70c809333e75fdcff2f6f1e838f29cfb872e1420a63cdf7f3a78
 
-# Add Containerpilot and its configuration
+RUN curl --retry 7 -Lso /tmp/consul-template.zip "https://releases.hashicorp.com/consul-template/${CONSUL_TEMPLATE_VERSION}/consul-template_${CONSUL_TEMPLATE_VERSION}_linux_amd64.zip" \
+    && echo "${CONSUL_TEMPLATE_SHA1}  /tmp/consul-template.zip" | sha256sum -c \
+    && unzip /tmp/consul-template.zip -d /usr/local/bin \
+    && rm /tmp/consul-template.zip
+
+# Add Containerpilot and set its configuration
 ENV CONTAINERPILOT_VER 2.0.1
```

Comment:
2.1.0 is out. Seems like whenever we update one of these we should make sure we're bumping the ContainerPilot version in the process just to keep them current.